assets/prototype/domain/eventEditor/data/queries/priceTypes/useDefaultPriceType.ts

```diff
@@ -1,15 +1,14 @@
 import isEmpty from 'ramda/src/isEmpty';
 import usePriceTypes from './usePriceTypes';
 import { PriceType } from '../../types';
+import { isDefaultPriceType } from '../../../../shared/predicates/priceTypes/selectionPredicates';
 
 /**
  * A custom react hook for retrieving the default price type object.
  */
 const useDefaultPriceType = (): PriceType => {
 	const allPriceTypes: PriceType[] = usePriceTypes();
-	const percentSurchargePriceTypes: PriceType[] = allPriceTypes.filter(
-		({ baseType, isPercent }) => baseType === 'SURCHARGE' && isPercent
-	);
```

Comment:
😱  uhhhh... doesn't look like that was returning default prices !!!
was this caught by adding tests? If so then 👍 

[EDIT]
oops.. my bad... was thinking of prices not price types